File: 1803/1803.cpp
class Solution {
public:
    int countPairsWithXorInARange(TreeNode* root) {
        if(!root) return 0;
        return 1+max(countPairsWithXorInARange(root->left),countPairsWithXorInARange(root->right));
    }
};

File: 1803/1803.cs
public class Solution {
    public int CountPairsWithXorInARange(TreeNode root) {
        if (root == null) return 0;
        return 1 + Math.Max(CountPairsWithXorInARange(root.left), CountPairsWithXorInARange(root.right));
    }
}